orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/ScmService.java

```diff
@@ -0,0 +1,25 @@
+package com.netflix.spinnaker.orca.igor;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Map;
+
+@Component
+@RequiredArgsConstructor
+public class ScmService {
```

Comment:
Turns out this ended up being just a thin wrapper around `igorService`. Not sure it's worth it, but I kept it to follow the pattern of `BuildService`.

Reply:
Hmm. I'd say, probably not worth it if you're not changing the api calls at all. I'd just wire in the `igorService` instead of creating this file.